test/data/base/04-op-in-define.yaml

```diff
@@ -0,0 +1,12 @@
+otk.version: "1"
+
+otk.define.name:
+  foo:
+    otk.op.join:
```

Comment:
A test with an `otk.join` outside a define would be nice too, maybe something like:
```yaml
diff --cc test/data/base/04-op-in-define.yaml
index 9a1143c,1f80deb..0000000
--- a/test/data/base/04-op-in-define.yaml
+++ b/test/data/base/04-op-in-define.yaml
@@@ -7,5 -7,12 +7,11 @@@ otk.define.name
          - - 1
          - - 2
  
- pipelines:
-   - foo: ${foo}
+ otk.target.osbuild.name:
+   pipelines:
+     - foo: ${foo}
 -
+   bar:
+     otk.join:
+       values:
+         - [3, 2, 1]
+         - [6, 5, 4]
diff --git a/test/data/base/04-op-in-define.json b/test/data/base/04-op-in-define.json
index dfab33c..afd6a8a 100644
--- a/test/data/base/04-op-in-define.json
+++ b/test/data/base/04-op-in-define.json
@@ -6,5 +6,8 @@
         2
       ]
     }
+  ],
+  "bar": [
+    1, 2, 3, 4, 5, 6
   ]
 }
```
(I hope this translates right)

Reply:
I've added this as a separate test case (`09-op.yaml`).